bin/cozy-light

```diff
@@ -59,7 +69,7 @@ program
   .action(actions.enable);
 
 program
-  .command('install-distro [distro]')
+  .command('install-distro <distro>')
```

Comment:
I prefer forget about it, they have 25 pending PRs, some i dont quiet understand why it s not yet merged. I dont want spend time to arg with them, seems too difficult, i just reset optionnal argument following your description of the behavior. thank you.